sdk/core/az_core/src/az_json_parser.c

```diff
@@ -43,13 +43,13 @@ az_json_parser_stack_last(az_json_parser const* json_parser)
 }
 
 AZ_NODISCARD AZ_INLINE az_result
-az_json_parser_push_stack(az_json_parser* json_parser, _az_json_stack json_stack)
+az_json_parser_push_stack(az_json_parser* ref_json_parser, _az_json_stack json_stack)
```

Comment:
@antkmsft please avoid making such SDK-wide renaming/refactoring changes to separate PRs outside of unrelated bug fixes/features. This way folks can more easily review and react to them :)

If you happen to touch a file already in a particular PR, then re-formatting or some renaming in that file is fine, but otherwise, I recommend splitting up those changes.